events/exchange/exchange_test.go

```diff
@@ -52,11 +52,8 @@ func TestExchangeBasic(t *testing.T) {
 	eventq2, errq2 := exchange.Subscribe(ctx2)
 
 	t.Log("publish")
-	var wg sync.WaitGroup
-	wg.Add(1)
 	errChan := make(chan error)
```

Comment:
Change is good. But I think we can add cap for `errChan` like, `errChan := make(chan error, 1)`, so that we don't need to worry about deadlock case. WDYT?

Reply:
@fuweid , yeah that would be another way to fix it.